### DISCUSSION TITLE
serial "null modem" between two tasmotas

### DISCUSSION THREAD

**Discussion by ouinouin** [arendst/Tasmota#18227] [Q&A]
Hi,  im fiddling with two M5 stickC ,  i m trying to send some characters from one to another one through serial (the final goal will be to send the acceletrometer to another one to compare.). I really  need to have this through a wire.

I ve setup GPIO 32 as serial tx and GPIO33 as serial RX on the M5stick 1 
I ve setup GPIO 33 as serial tx and GPIO32 as serial RX on the M5stick 2
i issues seriallog 2 on both units
im trying to send some char through the console with serialsend5 AABBCC, i ve seen nothing received.
did the same try with Sserialsend5 

iv been trying all sort of combinations with serial TX RX , serialb bridge tx rx, but didnt manalge to see anything reveived on unit 2.

some difficulties I have : setting GPIO 32 and 33 as serial tx and serial RX does make a software serial or a hardware serial (can the ESP32 uart be mapped on any pin ? 
does Sserial applies to the software serial port ? 
my config doesnt never explicitly have the serial uart linked to USB even while i autoconfigure it for the M5stickC at the beginning of the setup. , is it expected  ?  i m still able to receive the usual tasmota logs even if those pins are not declared. i never paid attention on the other units if there is a default TX RX pins setup or if its hardcoded somewhere and just moved if you explicitly set tx and rx to other pins (but still asking myself if its possible).

in summary i m abit confused : is my goal achievable : send some payload to another tasmota through serial.
is seriallog 2 enough to check if i m successful ?
